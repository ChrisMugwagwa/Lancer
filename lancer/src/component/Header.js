import React, {Component} from 'react';
import Navbar from 'react-bootstrap/Navbar';
import { Nav } from 'react-bootstrap';

export default class Header extends Component {
    render(){
        return(
            <Navbar collapseOnSelect expand='lg' bg='dark' variant='dark'>
                <Navbar.Brand>Lancer</Navbar.Brand>
                <Navbar.Collapse id='responsive-navbar-nav'>
                    <Nav.Link href="#home">Home</Nav.Link>
                    <Nav.Link href="#video">Video</Nav.Link>
                    <Nav.Link href="#photography">Photography</Nav.Link>
                    <Nav.Link href="#gdesign">Graphic Design</Nav.Link>
                    <Nav.Link href="#register">Register</Nav.Link>
                </Navbar.Collapse>
            </Navbar>
        );
    }
}
