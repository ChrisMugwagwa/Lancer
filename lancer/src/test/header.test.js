import React from 'react';
import {render, cleanup} from '@testing-library/react';
import renderer from 'react-test-renderer';
import Header from '../component/Header'

afterEach(cleanup);

it ("Displays the header",() =>{
    const tree = renderer.create(<Header></Header>)
    expect(tree).toMatchSnapshot();
});