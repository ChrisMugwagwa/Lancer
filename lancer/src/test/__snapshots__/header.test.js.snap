// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays the header 1`] = `
<nav
  className="navbar navbar-expand-sm navbar-dark bg-dark"
  id="header"
>
  <span
    className="navbar-brand"
  >
    Lancer
  </span>
  <div
    aria-expanded={null}
    className="navbar-collapse collapse"
    id="responsive-navbar-nav"
  >
    <a
      className="nav-link"
      href="/"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      Home
    </a>
    <a
      className="nav-link"
      href="/video"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      Video
    </a>
    <a
      className="nav-link"
      href="/photography"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      Photography
    </a>
    <a
      className="nav-link"
      href="/gdesign"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      Graphic Design
    </a>
    <a
      className="nav-link"
      href="/register"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      Register
    </a>
  </div>
  <button
    className="btn btn-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Login
  </button>
</nav>
`;
